src/settings/MappingProfiles/detailsSections/view/OrderDetailsSection/EResourcesDetails.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedDate,
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  Accordion,
+  Col,
+  Row,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  getFieldValue,
+  getFieldValueByPath,
+  renderCheckbox,
+} from '../../utils';
+import { mappingProfileFieldShape } from '../../../../../utils';
+
+export const EResourcesDetails = ({ mappingDetails }) => {
+  const accessProvider = getFieldValue(mappingDetails, 'accessProvider', 'value');
+  const activated = getFieldValue(mappingDetails, 'activationStatus', 'booleanFieldAction');
+  const activationDue = getFieldValue(mappingDetails, 'activationDue', 'value');
+  const createInventory = getFieldValueByPath(mappingDetails, 'order.poLine.eresource.createInventory', 'value');
+  const materialType = getFieldValueByPath(mappingDetails, 'order.poLine.eresource.materialType', 'value');
+  const trial = getFieldValue(mappingDetails, 'trial', 'booleanFieldAction');
+  const expectedActivation = getFieldValue(mappingDetails, 'expectedActivation', 'value');
+  const userLimit = getFieldValue(mappingDetails, 'userLimit', 'value');
+  const resourceUrl = getFieldValue(mappingDetails, 'resourceUrl', 'value');
+
+  const activatedCheckbox = renderCheckbox('order.eResourcesDetails.activated', activated);
+  const trialCheckbox = renderCheckbox('order.eResourcesDetails.trial', trial);
+
+  const formattedActivationDue = (
+    <FormattedDate
+      value={activationDue}
+      day="2-digit"
```

Comment:
Could you please investigate possible to move `day="2-digit" month="2-digit" year="numeric"` to variable and reuse it?